docs/software_setup/sd_card.md

```diff
@@ -3,94 +3,86 @@
    under the module)
 
 2. Connect the monitor, keyboard, and mouse to the Nano
-3. Power on the Jetson Nano by connecting the micro USB charger to the micro USB port
+3. Power on the Jetson Nano by connecting the micro USB (for Jetson Nano (4GB)) or USB-C (for Jetson Nano 2GB) charger to the port
 
     ???+ attention 
         We recommend first booting the Jetson Nano once without the piOLED / motor driver connected.
 
         This way you can check to make sure the system boots properly from the SD card image without worrying about hardware issues.  After you've verified that it boots, reconnect the piOLED, double check your wiring, and boot again.
 
-## Step 3 - Connect JetBot to WiFi
+## Step 4 - Connect JetBot to WiFi
 
-1. Log in using the user ``jetbot`` and password ``jetbot``
+Next you'll need to connect to WiFi.  To reduce memory consumption, we disable the Ubuntu GUI in the latest JetBot SD card image.  For this reason, you'll need to use the command line to connect to WiFi.
 
-2. Connect to a WiFi network using the Ubuntu desktop GUI
+1. Log in using the user ``jetbot`` and password ``jetbot``
+    
+2. Connect to a WiFi network using the following command
 
+    ```bash
+    sudo nmcli device wifi connect <SSID> password <PASSWORD>
+    ```
+    
 Your Jetson Nano should now automatically connect to the WiFi at boot and display it's IP address on the piOLED display.
 
 ???+ tip
     If you're having trouble figuring out how to get connected to Wi-Fi, check out the [Wi-Fi setup](wifi_setup.md) page for more detailed instructions 
 
-## Step 4 - Connect to JetBot from web browser
+## Step 5 - Connect to JetBot from web browser
 
-After your robot is connected to WiFi, you can connect to the robot from a web browser by performing the following steps
+After your robot is connected to WiFi, you no longer need to have the robot connected by a monitor.  You can connect to the robot from your laptop's web browser by performing the following steps
 
-1. Shutdown JetBot using the Ubuntu GUI
+1. Shutdown JetBot using the command line
+
+    ```bash
+    sudo shutdown now
+    ```
 
 2. Unplug your HDMI monitor, USB keyboard, mouse and power supply from Jetson Nano
+
 3. Power the JetBot from the USB battery pack by plugging in the micro-USB cable
 4. Wait a bit for JetBot to boot
 2. Check the IP address of your robot on the *piOLED* display screen.  Enter this in place of ``<jetbot_ip_address>`` in the next command
-3. Navigate to ``http://<jetbot_ip_address>:8888`` from your desktop's web browser
+3. Navigate to ``http://<jetbot_ip_address>:8888`` from your desktop's web browser. You can do this from any machine on your local network.  
+4. Sign in using the password ``jetbot``.
 
-???+ info
-    You shouldn't need to connect your robot to a monitor past this step! 
+That's it, you've now accessed JetBot's remote programming environment! 
 
-## Step 5 - Install latest software (optional)
```

Comment:
Since this step no longer work, instructions for Waveshare jetbot got broken: https://www.waveshare.com/wiki/JetBot_AI_Kit#User_Guides

They maintain their own fork with some overrides to support their own hardware: https://github.com/waveshare/jetbot

Without those overrides e.g motors does not work.